frontend/common/data/base/_data.js

```diff
@@ -77,6 +78,13 @@ module.exports = {
             options.headers.AUTHORIZATION = `Token ${this.token}`;
         }
 
+        let url = _url;
```

Comment:
This file is where all http requests go through, in this case we want to replace credential URLs with a normal URL since browsers are dropping support for them.

Task: 

1 - Turn https://foo:bar3@httpbin.org/basic-auth/foo/bar3 into https://httpbin.org/basic-auth/foo/bar3
2 - If URLs do not contain credentials, treat them as a standard URL.
2 - Parse the credential in the URL and set an authorisation header with base 64 of username:password